.github/workflows/webpage.yml

```diff
@@ -35,10 +38,27 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: aiida-registry fetch
-    - name: make pages
-      run: aiida-registry html
-    - name: Deploy 🚀
-      uses: JamesIves/github-pages-deploy-action@4.1.1
+    - name: Commit json file
```

Comment:
we may eventually need/want to decouple the fetching of the data from building the web site (`aiida-registry fetch` is fast, but the `test-install` workflow that collects all of the info is necessarily slow).

until we've added the data collection, this approach is ok though

Reply:
Thanks @ltalirz for reviewing,
should this be done by separating the data collection and website building into two jobs?

Reply:
yes, perhaps in the future - not needed here though, all good